requirements-dev.txt

```diff
@@ -1,3 +1,7 @@
 flake8
 pytest
 vincent
+jupyter_client
+ipykernel
+nbconvert
+selenium
```

Comment:
Already add `@knil-sama` here for you :wink: 

Hopefully you will be able to reabse #18 once this is merged and everything will pass. Although selenium may need some config options and I am not familiar with it.